examples/prisma-react/.runTest.ts

```diff
@@ -0,0 +1,37 @@
+import { page, run, urlBase, autoRetry, fetchHtml } from '../../libframe/test/setup'
+
+export { runTest }
+
+function runTest(cmd: 'npm run test:dev' | 'npm run test:prod') {
+  run(cmd)
+
+  test('Add to-do item', async () => {
+    await page.goto(`${urlBase}/`)
+    expect(await page.textContent('body')).toContain('todo list')
+    await page.fill('input[placeholder="Title"]', 'title')
+    await page.fill('input[placeholder="Content"]', 'content')
+    await page.click('button[type="submit"]')
+
+    await autoRetry(async () => {
+      expect(await page.textContent('body')).toContain('title')
+      expect(await page.textContent('body')).toContain('content')
+    })
```

Comment:
It's somewhat confusing to fill the title input with "Title". Maybe we can make it something more realistic like "Cherries" + "Buy cherries".